src/js/tabs/osd.js

```diff
@@ -1968,15 +1965,13 @@ OSD.updateDisplaySize = function() {
     if (videoType === 'AUTO') {
         videoType = 'PAL';
     }
```

Comment:
One question. We have a way to know if there is an HD system configured/attached? I don't have the device, so I'm not sure if we have this information. In this case it will be interesting to select HD and not PAL here.

Reply:
What I could do is change the firmware so that if a `MSP_SET_OSD_CANVAS` command is received, the `vcd_video_system` could be set to `HD`. If a save is done subsequent to that with the VTX connected, then both that and the canvas size would be preserved and used even if the VTX isn't powered.

See https://github.com/betaflight/betaflight/pull/11964/commits/fa11425550bc385c7959c1a056b5b6f7e6703944

Reply:
Is not enough if MSP+Displayport is selected? Is simply a best effort, we let the user modify it if needed.

Reply:
MSP VTX and displayport are one combined option and don’t necessarily mean HD.